### PR TITLE
Some fixes to the OSSM docs

### PR DESCRIPTION
**Pull request by brian-avery**
@neal-timpe @JStickler Please take a look. These are some failures picked up by the Maistra linter/link checker.

There are also several anchors that the OSSM docs link to but don't exist (pages exist, just the anchors don't). Will add those in either another commit or separate PR, but wanted to start with this.

### CODE REVIEW

File: modules/ossm-routing.adoc [hunk 1]
```diff
@@ -218,7 +218,7 @@ A service entry adds an entry to the service registry that {ProductName} maintai
 * Define retry, timeout, and fault injection policies for external destinations.
 * Run a mesh service in a Virtual Machine (VM) by adding VMs to your mesh.
 * Logically add services from a different cluster to the mesh to configure a multicluster {ProductName} mesh on Kubernetes.
-* You don’t need to add a service entry for every external service that you want your mesh services to use. By default, {ProductName} configures the Envoy proxies to passthrough requests to unknown services. However, you can’t use {ProductName} features to control the traffic to destinations that aren’t registered in the mesh.
+* You don’t need to add a service entry for every external service that you want your mesh services to use. By default, {ProductName} configures the Envoy proxies to pass through requests to unknown services. However, you can’t use {ProductName} features to control the traffic to destinations that aren’t registered in the mesh.
```
Comment (neal-timpe) [inline modules/ossm-routing.adoc:221]: @brian-avery 

Instead of 
`to pass through requests`
Would it be clearer to say 
`to pass requests through`

?

Comment (JStickler) [reply modules/ossm-routing.adoc:221]: I'm not sure if "passthrough" isn't an Envoy feature?  I can find this spelled as one word in a couple of places in the Istio docs.

https://istio.io/latest/docs/tasks/traffic-management/egress/egress-control/#envoy-passthrough-to-external-services 

https://istio.io/latest/blog/2019/monitoring-external-service-traffic/#what-are-blackhole-and-passthrough-clusters

Comment (brian-avery) [reply modules/ossm-routing.adoc:221]: Passthrough is an Envoy feature, but this is referring to the ability to route requests through Envoy. In terms of Envoy, `passthrough` is a specific type of cluster where all traffic entering Envoy is allowed to transit through. This is referring to somethign different. Regarding "to pass requests through", that makes sense to me.

**Review by neal-timpe — APPROVED** [review]

**Pull request merged**